packages/template-retail-react-app/app/utils/address-suggestions.js

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const COUNTRY_CODE_MAP = {
+    USA: 'US',
+    Canada: 'CA'
+}
```

Comment:
Should this pull from the locales in `config/sites.js`?

I image that other projects with other countries will be updating this and the supported countries will be different depending on the site.

Reply:
Thanks for reminding me of that! That was a while ago. 

Since there was precedent for having hard coded country codes before this work, I won't treat this as a blocker for this release.

Is there a follow up ticket for eventually cleaning up the hard coded US / CA?